### DISCUSSION TITLE
Possible Bug, Hyprland might be preventing me from logging into some x11 window managers.

### DISCUSSION THREAD

**Discussion by greg086** [hyprwm/Hyprland#8887] [General]
Not posting in issues yet because I'm not completely sure if this has to do with hyprland itself yet.

Hi, I ran into an issue that started the 1st day I tried Hyprland about a month and a half ago. I was messing around in the compositor and then tried to log back into qtile, my x11 window manager and was kicked back to the sddm login screen. Same think with some other window managers (awesome and i3, but not herbstluft or desktop environments).

I figured out that it was something wrong with my home directory because qtile worked on a fresh linux install with a fresh home but as soon as I mounted my home drive as /home in the fstab, it stopped working. I initially fixed the issue with a reinstall of linux and a fresh home and using my home drive for data/games. About 2 weeks later qtile stopped working again. After another fresh install I've stayed away from Hyprland for the past few weeks.

I'm now wondering what files in the home directory Hyprland writes to. I also have a hunch that this is specific to my setup so I'll share my specs.
CPU: Ryzen 2700x
GPU: RTX 2070 Super
32GB Ram
2 4K 60 Hz monitors
OS: Happened on Arco and Endeavor (I'm on mainline arch now but probably doesn't matter since the other two are arch based)

Thanks in advance.

**Comment by vaxerski** [comment]
I doubt it's something hyprland is doing, as we don't really do much when we are _literally not running_, so it's probably some misconfiguration on your end.

**Reply by greg086** [reply]
Both times it happened, it was immediately after logging out of hyprland. That's why I want to know where in the home directory hyprland can write to (while running). Is it just the log file? If so it's definitely something else causing it.

**Reply by vaxerski** [reply]
log directory, ~/.local/share/hyprland and config file, that's it

**Reply by greg086** [reply]
Thank you, I can rule hyprland out now.
